myuw/templates/handlebars/card/accounts/tuition.html

```diff
@@ -3,115 +3,133 @@
-                                <h4 class="card-badge-label">Account Credit</h4>
-                            </div>
-                            <div class="pull-right">
-                                <span class="card-badge-value">+${{plain_tuition}} CR</span>
-                            </div>
-                        </div>
-                    </li>
-
-        {{else}}  <!-- else, there is a balance -->
-
-		    	<li>
-	                <div class="card-badge clearfix" style="padding-bottom:0">
-	                    <div class="pull-left">
-	                        <h4 class="card-badge-label">You owe</h4>
-	                    </div>
-						<div class="pull-right">
-	                        <span class="card-badge-value text-danger">${{tuition_accbalance}}</span>
-	            		</div>
+	        <li>
+	            <div class="card-badge clearfix">
+	                <div class="pull-left">
+	                    <h4 class="card-badge-label">Account Credit<br><span style="font-size:14px;font-weight:normal;line-height:32px;">Student Fiscal Services</span></h4>
 	                </div>
+	                <div class="pull-right">
+	                    <span class="card-badge-value">+${{plain_tuition}} CR</span>
+	                    <span class="card-badge-action pull-right" style="clear:both;margin-top:-6px;">No payment needed<br></span>
+	                    <span class="card-badge-action pull-right" style="clear:both;" ><a href="https://sdb.admin.uw.edu/sisStudents/uwnetid/tuition.aspx" target="_blank" data-linklabel="Tuition Statement">Tuition Statement</a></span>
 
-
-	            {{#if tuition_date}}
-
-	                <div class="tuition-due card-badge clearfix middle-row">
-	                    <div class="pull-left">
-	                        <h4 class="card-badge-label">Tuition due </h4>
-	                    </div>
-						<div class="pull-right">
-	                        <span class="card-badge-value" aria-hidden="true">{{toFriendlyDate tuition_due}}</span>
-							<span class="sr-only">{{toFriendlyDateVerbose tuition_due}}</span>
-			        &nbsp;
-	                        <span class="card-badge-action pull-right">in {{tuition_date_offset}} days</span>
-	                    </div>
 	                </div>
+	            </div>
+	        </li>
+
+        {{else}}  <!-- else, there is a balance -->
 
-	            {{/if}}
-                <div class="card-badge clearfix middle-row">
-                    <div class="pull-right" >
-                        <div class="btn btn-default card-action-btn"><a href="http://f2.washington.edu/fm/sfs/tuition/payment" target="_blank" data-linklabel="Make Tuition Payment">Make payment</a></div>
+	    	<li>
+                <div class="card-badge clearfix" style="padding-bottom:0">
+                    <div class="pull-left">
+                        <h4 class="card-badge-label">Amount Due <br><span style="font-size:14px;font-weight:normal;line-height:32px;">Student Fiscal Services</span></h4>
```

Comment:
`<span style="font-size:14px;font-weight:normal;line-height:32px;">`

This inline css appears a few times in this file and should probably be refactored into a CSS class.